project1/src/components/Comments/Comments.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "../../App.css";
+import { action } from "@storybook/addon-actions";
+
+import { storiesOf } from "@storybook/react";
+import Comments from "./Comments";
+import UserComment from "./UserComment";
+
+storiesOf("Comments", module)
+  .add("Comments", () => (
+    <Comments
+      comments={[
```

Comment:
move the comments to a file called fixtures, better for readability and to reuse it in another stories